src/components/Layout/ActivityBar/ActivityBar.tsx

```diff
@@ -31,22 +36,34 @@ export function ActivityBar() {
             width="32"
           />
         </Flex>
-        <Grid gap="5" mt="4">
+        <Grid
+          css={css`
+            flex: 1 1 0;
+          `}
+          gap="5"
+          mt="4"
+        >
           <NavIconButton
             to={getRoutePath('home')}
             icon={<HomeIcon />}
             tooltip="Home"
           />
         </Grid>
 
-        <Flex direction="column" align="center" gap="3" mt="auto">
+        <Flex direction="column" align="center" gap="3">
           <ThemeSwitcher />
           <ProxyStatusIndicator />
           <SettingsButton />
           <HelpButton />
+        </Flex>
+        <Separator orientation="horizontal" size="2" />
```

Comment:
I'm letting the `Separator` be, otherwise I'd need to implement some kind of `<NotFeature />` component. 😅

Reply:
```suggestion
        <Feature feature="cloud-auth">
          <Separator orientation="horizontal" size="2" />
        </Feature>
```